src/SDKs/CognitiveServices/dataPlane/Knowledge/QnAMaker/QnAMaker/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "cognitiveservices/data-plane/QnAMaker" -AutoRestVersion "latest"
```

Comment:
Was the code generated using this script? 
There should be 2 additional files generated, a .txt and a SdkInfo_*.cs please commit these files

Reply:
done